orchestration/templates/ingest-clinvar-archive.yaml

```
@@ -120,7 +120,21 @@ spec:
                   value: '{{ $extractionPvc }}'
                 - name: file-path
                   value: 'ClinVarVariationRelease/part-1.json'
-          # 6) Run Dataflow on uploaded JSON.
+          # 6) Stage a release_history object in GCS
+          - name: stage-release-history
+            dependencies: [extract-release-date]
+            templateRef:
+              name: create-json-object
+              template: main
+            arguments:
+              parameters:
+                - name: release-date
+                  value: '{{ "/{{tasks.extract-release-date.outputs.result}}.xml" }}'
```

This should just be the release date, without the file-system stuff:
```suggestion
                  value: '{{ "{{tasks.extract-release-date.outputs.result}}" }}'
```